__tests__/controllers/users.controller.spec.ts

```diff
@@ -48,6 +48,8 @@ describe('requesting all users', () => {
     expect(response.status).toBe(200);
   });
 
+  // prettier-ignore-start
```

Comment:
is there a general prettier/eslint rule that we can apply to escape quotes? so we can save the need for those comments